Erro ao executar npm run seed:default

## Feature Description

Ajustes no Defuault da seed.

Só consegui rodar a seed sem o default.

## Acceptance criteria

Apresentou o seguinte erro:

```
C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api>npm run seed:default

> adonis-api-app@4.1.0 seed:default C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api
> node ace seed --files='0_RoleSeeder.js, 2_TaxonomySeeder.js, 3_TermSeeder.js, 5_PermissionSeeder.js'


TypeError: Cannot read property 'name' of undefined


1 anonymous
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\@adonisjs\ace\src\Command\index.js:527

2 anonymous
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\lodash\lodash.js:13806

3 arrayEach
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\lodash\lodash.js:516

4 Function.transform
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\lodash\lodash.js:13805

5 Function.commanderAction
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\@adonisjs\ace\src\Command\index.js:526

6 Command.listener
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\commander\index.js:315

7 Command.parseArgs
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\commander\index.js:651

8 Command.parse
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\commander\index.js:474

9 Kernel.invoke
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\@adonisjs\ace\src\Kernel\index.js:329

10 Ignitor._invokeAce
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\@adonisjs\ignitor\src\Ignitor\index.js:532

11 Ignitor.fireAce
  C:\Users\migue\Desktop\UFERSA\plataforma-sabia\packages\api\node_modules\@adonisjs\ignitor\src\Ignitor\index.js:847

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! adonis-api-app@4.1.0 seed:default: `node ace seed --files='0_RoleSeeder.js, 2_TaxonomySeeder.js, 3_TermSeeder.js, 5_PermissionSeeder.js'`

```

--- a/packages/api/package.json
+++ b/packages/api/package.json
@@ -15,7 +15,7 @@
     "migration:run": "node ace migration:run",
     "migration:refresh": "node ace migration:refresh",
     "seed": "node ace seed",
-    "seed:default": "node ace seed --files='0_RoleSeeder.js, 2_TaxonomySeeder.js, 3_TermSeeder.js, 5_PermissionSeeder.js'"
+    "seed:default": "node ace seed --files=\"0_RoleSeeder.js, 2_TaxonomySeeder.js, 3_TermSeeder.js, 5_PermissionSeeder.js\""
   },
   "keywords": [
     "adonisjs",

